backend/inspirehep/oai/utils.py

```diff
@@ -37,11 +37,15 @@ def has_cern_affiliation(record):
     """Check if the record should be part of `CERN:arXiv` set."""
     affiliations_values = get_value(record, "authors.affiliations.value", default=[])
     affiliations = chain.from_iterable(affiliations_values)
-    return any(
+    authors = get_value(record, "authors", default=[])
+    if any(
         affiliation
         for affiliation in affiliations
         if affiliation.lower().startswith("cern")
-    )
+    ) & any(author for author in authors if author["curated_relation"]):
```

Comment:
this is causing an error in tests cause not all authors have curated_relation. Instead you have to do `author.get('curated_relation`),